src/main/java/com/robosh/ejournal/data/entity/admin/Admin.java

```diff
@@ -26,7 +27,7 @@
 public class Admin {
 
     @Id
-    @GeneratedValue
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
if we are using Hibernate 5 it is better to:
 @GeneratedValue(
    strategy= GenerationType.AUTO, 
    generator="native"
)
https://vladmihalcea.com/why-should-not-use-the-auto-jpa-generationtype-with-mysql-and-hibernate/

Reply:
Hmm, interesting. I will change it